core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -159,10 +178,11 @@ private[spark] class AppStatusListener(
       details.getOrElse("Spark Properties", Nil),
       details.getOrElse("Hadoop Properties", Nil),
       details.getOrElse("System Properties", Nil),
-      details.getOrElse("Classpath Entries", Nil))
+      details.getOrElse("Classpath Entries", Nil),
+      Nil)
 
-    coresPerTask = envInfo.sparkProperties.toMap.get(CPUS_PER_TASK.key).map(_.toInt)
-      .getOrElse(coresPerTask)
+    defaultCoresPerTask = envInfo.sparkProperties.toMap.get(CPUS_PER_TASK.key).map(_.toInt)
```

Comment:
Nit: Sometimes the code talks about cores and sometimes about CPUs and within this line we have a conversion between the two. Is our terminology correct?

Reply:
yeah unfortunately the entire code base isn't consistent on that. its really cpus since its per task, but left it named similar. I can update.